lib/utilities/is-project.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = function isProject(obj) {
```

Comment:
also mentioned by @brendenpalmer, but lets move to directly checking if `obj === this.project` (the project the per-addon-bundle-cache was created for).

Reply:
That removes the last use of isBundleRoot, so removed that. Took the contents of this function and moved it to PackageInfo.isForProject, which is the only place isProject was used after isBundleRoot was removed.